iModelCore/iModelPlatform/Tests/DgnProject/Compatibility/IModelCompatibilityTests.cpp

```diff
@@ -130,7 +130,19 @@ void Assert_BuiltinSchemaVersions_2_0_0_0 (TestIModel& testDb)
 //+---------------+---------------+---------------+---------------+---------------+------
 void Assert_BuiltinSchemaVersions_2_0_0_1(TestIModel& testDb)
     {
-    EXPECT_EQ(8, testDb.GetSchemaCount()) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("CoreCustomAttributes") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("ECDbMap") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("ECDbFileInfo") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("ECDbMeta") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("ECDbSystem") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("ECDbSchemaPolicies") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("Generic") != nullptr) << testDb.GetDescription();
+    EXPECT_TRUE(testDb.GetDb().Schemas().GetSchema("BisCore") != nullptr) << testDb.GetDescription();
+
+    // BisCustomAttributes schema is added to the test files containing BisCore schema having reference schema BisCustomAttributes (e.g BisCore.1.0.16)
+    if (testDb.GetSchemaVersion("BisCore") == SchemaVersion(1, 0, 16))

```

Comment:
Should we do this test if the version is greater than or equal to 1.0.16?

Reply:
If we are sure that this reference schema will not be removed from BisCore schema in future then we can add this check. Otherwise, we may face similar situation in future that we have to disable test in old test runner.

Reply:
I have updated this check.